action/report/htmlreport.tpl

```diff
@@ -78,27 +78,42 @@
               </tr>
             </thead>
             <tbody>
-                {{- range $ind, $slo := .Result.Insights.SLOs }}
+                {{- range $ind, $slo := .Result.Insights.SLOs.Upper }}
```

Comment:
This change enables users to specify both upper and lower limits for SLOs very easily. Previous template only supported upper.